### ISSUE TITLE
Smoothing fails for constant scalars

### ISSUE THREAD

**Issue by Dapid** [tensorflow/tensorboard#1496]
I have a network training where the learning rate didn't change:
![base](https://user-images.githubusercontent.com/1616942/46606313-bdd19b00-cafc-11e8-92bd-205d1cecb977.gif)

When applying smoothing, I get the following artefacts:
![smooth](https://user-images.githubusercontent.com/1616942/46606338-ebb6df80-cafc-11e8-8653-cf007e6ab08d.gif)

Note that when pushing smoothing all the way to 1 the line disappears completely, since it gets smoothed down to 0.

More info:
TensorBoard 1.10.0 and 1.11.0. (Screenshots correspond to the former).
Linux Fedora 27
Python 3.6
Observed in both Firefox and Chrome

**Comment by nfelt** [comment]
Duplicate of #786

**Comment by nfelt** [comment]
Yes, unfortunately this is a known issue with the current smoothing.  Please follow #786.
